pycaret/internal/tabular.py

```diff
@@ -10615,6 +10615,72 @@ def create_docker(
         """
     )
 
+def create_app(estimator, app_kwargs):
+    """
+    This function creates a basic gradio app for inference.
+    It will later be expanded for other app types such as 
+    Streamlit.
+
+
+    Example
+    -------
+    >>> from pycaret.datasets import get_data
+    >>> juice = get_data('juice')
+    >>> from pycaret.classification import *
+    >>> exp_name = setup(data = juice,  target = 'Purchase')
+    >>> lr = create_model('lr')
+    >>> create_app(lr)
+
+
+    estimator: scikit-learn compatible object
+        Trained model object
+
+
+    app_kwargs: dict, default = {}
+        arguments to be passed to app class.
+
+
+    Returns:
+        None
+    """
+
+    import gradio as gr
+
+    all_inputs = []
+
+    data_without_target = get_config('data_before_preprocess').copy()
+    target_name = get_config('prep_pipe')[0].target
+    data_without_target.drop(target_name, axis=1, inplace=True)
+    
+    try:
+        for i in get_config('prep_pipe')[0].features_todrop:
+            data_without_target.drop(i, axis=1, inplace=True)
+    except:
+        pass
+
+    try:
+        for i in get_config('prep_pipe')[0].id_columns:
+            data_without_target.drop(i, axis=1, inplace=True)
+    except:
+        pass
```

Comment:
Should this try catch block be outside or inside the loop?

Reply:
there is also `errors="ignore"` param for `drop`

Reply:
furthermore, drop can take in a list, so that's what we should be using here for better performance, instead of doing this in a loop